.github/workflows/build.yml

```diff
@@ -108,6 +111,9 @@ jobs:
             FEATURES="${FEATURES},standard,pqc"
           elif [ "${{ matrix.name }}" = "log" ]; then
             FEATURES="${FEATURES},standard,log"
+          elif [ "${{ matrix.name }}" = "no_sha1" ]; then
+            #nssdb doesn't work with no_sha1
```

Comment:
nit: Is the comment needed here? Would not it be better in the "exclude" part above, where the db is actually excluded from the matrix?

Reply:
Fixed, thanks!